### PR TITLE
Implement support for public updates channels

### PR DESCRIPTION
**Pull request by Nick-kel**
## Pull Request Etiquette

- [X] I have checked the PRs for upcoming features/bug fixes.
- [X] I have read the [contributing guidelines][contributing].

### Changes

- [X] Internal code
- [X] Library interface (affecting end-user code) 
- [X] Documentation

## Description

This PR adds Guild#setPublicUpdatesChannel and Guild#getPublicUpdatesChannel which are used by Discord to send notices to admins and moderators of Community guilds

### CODE REVIEW

**Comment by MinnDevelopment** [comment]
Duplicate of #1464

**Comment by Nick-kel** [comment]
@MinnDevelopment in the Discord api the field isn't named "community update channel" (like in your linked PR), it's named "public updates channel" (like in my PR, see https://discord.com/developers/docs/resources/guild)

**Comment by Andre601** [comment]
> @MinnDevelopment in the Discord api the field isn't named "community update channel" (like in your linked PR), it's named "public updates channel" (like in my PR, see https://discord.com/developers/docs/resources/guild)

It's still the same. It's just a different name in the Library.

File: src/main/java/net/dv8tion/jda/api/entities/Guild.java [hunk 1]
```diff
@@ -573,6 +573,17 @@ default int getMaxEmotes()
     @Nullable
     VoiceChannel getAfkChannel();
 
+    /**
+     * Provides the {@link net.dv8tion.jda.api.entities.TextChannel TextChannel} that has been set as the channel
+     * <br>If no public updates channel has been set as the AFK channel, this returns {@code null}.
+     * <p>
+     * This value can be modified using {@link GuildManager#setPublicUpdatesChannel(TextChannel)}.
+     *
+     * @return Possibly-null {@link net.dv8tion.jda.api.entities.VoiceChannel VoiceChannel} that is the AFK Channel.
```
Comment (arynxd) [inline src/main/java/net/dv8tion/jda/api/entities/Guild.java:582]: Shouldnt this, `{@link net.dv8tion.jda.api.entities.VoiceChannel VoiceChannel} that is the AFK Channel.`, become `{@link net.dv8tion.jda.api.entities.TextChannel TextChannel} that is the Public Updates Channel.`,

**Pull request closed without merging**